### PR TITLE
feat: Update Argo Chart to support v2.6.1

### PR DESCRIPTION
**Pull request by paguos**
- Update Workflow CRD
- Add CronWorkflow CRD
- Replace argo-ui with argo-server
- Add cronworkflows to aggregate-roles
- Add cronworkflows to controller-clusterrole
- Add create workflow rights to controller clusterrole
- Bump chart version

Checklist:

* [x] I have update the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] I have followed the testing instructions in the [contributing guide](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md).
* [x] I have signed the CLA and the build is green.
* [x] I will test my changes again once merged to master and published.

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo/crds/cron-workflow-crd.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (sudermanjr) [inline charts/argo/crds/cron-workflow-crd.yaml]: I think the other CRDs have an `if` here on the installCRD value in order to disable this for those who don't like helm CRD hooks

**Comment by simster7** [comment]
Could you also bump it to `2.6.1` as that is our latest release?

@benjaminws Please take a look once it's at 2.6.1

**Comment by paguos** [comment]
> Could you also bump it to `2.6.1` as that is our latest release?
> 
> @benjaminws Please take a look once it's at 2.6.1

Done 👍

**Comment by paguos** [comment]
@simster7 can we merge this?

**Pull request merged**